app/services/provider_details_creator.rb

```diff
@@ -31,10 +32,12 @@ def should_clear_selected_office?
   end
 
   def firm
-    Firm.find_or_create_by!(ccms_id: firm_id).tap do |firm|
+    current_firm = Firm.find_or_create_by!(ccms_id: firm_id).tap do |firm|
       firm.update!(name: firm_name)
       firm.offices << offices
     end
+    current_firm.permissions << @passported_permission unless current_firm.permissions.include?(@passported_permission)
```

Comment:
To me this says 'every provider has a a firm with passported permissions.'

This may cause a problem for when we move to non-passported. Maybe have a TODO comment or refactor to say
if there are no permissions on the firm then default to passported_permission 

`current_firm.permissions << @passported_permission if current_firm.permissions.empty?`